util/network-devp2p/src/ip_utils.rs

```diff
@@ -225,28 +225,13 @@ mod getinterfaces {
 				let sa: *const sockaddr_in = sa as *const sockaddr_in;
 				let sa = unsafe { &*sa };
 				let (addr, port) = (sa.sin_addr.s_addr, sa.sin_port);
-				(IpAddr::V4(Ipv4Addr::new(
-					(addr & 0x0000_00FF) as u8,
-					((addr & 0x0000_FF00) >>  8) as u8,
-					((addr & 0x00FF_0000) >> 16) as u8,
-					((addr & 0xFF00_0000) >> 24) as u8)),
-					port)
+				(IpAddr::V4(Ipv4Addr::from(addr)), port)
```

Comment:
Seems like we're encoding the octets in reverse order compared to `std`.

Reply:
Ok, I will check but I assumed the tests actually tested this?!